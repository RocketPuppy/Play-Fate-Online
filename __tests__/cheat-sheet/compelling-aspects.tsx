import * as React from "react";
import * as renderer from "react-test-renderer";
import CompellingAspects from "../../src/cheat-sheet/compelling-aspects";

test("renders all the pieces", () => {
  const render = renderer.create(<CompellingAspects />);

  expect(render.toJSON()).toMatchSnapshot();
});
