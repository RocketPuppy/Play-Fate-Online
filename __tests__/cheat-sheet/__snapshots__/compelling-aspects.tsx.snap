// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Compelling Aspects
  </h2>
  <p>
    Accept a complication for a fate point.
  </p>
  <dl>
    <dt>
      Event-based
    </dt>
    <dd>
      You have ____ aspect and are in ____ situation, so it makes sense that, unfortunately, ____ would happen to you. Damn your luck.
    </dd>
    <dt>
      Decision-based
    </dt>
    <dd>
      You have ____ aspect in ____ situation, so it makes sense that youâ€™d decide to ____. This goes wrong when ____ happens.
    </dd>
  </dl>
</div>
`;
